### PR TITLE
crowcpp-crow: add version 1.1.1, reomve older versions

### PR DESCRIPTION
**Pull request by toge**
Specify library name and version:  **crowcpp-crow/***

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by uilianries — DISMISSED** [review]
LGTM

**Review by uilianries — DISMISSED** [review]
LGTM

File: recipes/crowcpp-crow/all/conanfile.py [hunk 1]
```diff
@@ -42,7 +39,10 @@ def layout(self):
         cmake_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("boost/1.83.0")
+        if Version(self.version) < "1.1.0":
+            self.requires("boost/1.83.0")
+        else:
+            self.requires("asio/1.29.0", transitive_headers=True)
         if self.version == "0.2":
             self.requires("openssl/[>=1.1 <4]")
```
[the remark this answers is not in the document]
Comment (toge) [reply recipes/crowcpp-crow/all/conanfile.py:47]: @ericLemanissier 
Thank a lot!

**Review by ericLemanissier — APPROVED** [review]

**Review by RubenRBS — APPROVED** [review]

**Comment by ericLemanissier** [comment]
@toge 1.1.1 disappeared from https://github.com/CrowCpp/Crow/
Do you have some info ?

**Comment by toge** [comment]
@ericLemanissier 

Thanks a lot!
I create another PR for it.
https://github.com/conan-io/conan-center-index/pull/22637

**Pull request merged**